ci/jobs/python_requirements/get_metviewer.sh

```diff
@@ -0,0 +1,33 @@
+#! /bin/bash
+
+# set environment variables needed by METviewer docker-compose.yml
+export METVIEWER_DATA=$RUNNER_WORKSPACE
+export MYSQL_DIR=$RUNNER_WORKSPACE/mysql
+export METVIEWER_DIR=$RUNNER_WORKSPACE/output/metviewer
+export METVIEWER_DOCKER_IMAGE=dtcenter/metviewer
```

Comment:
This will pull the Docker image for dtcenter/metviewer:latest which is the latest stable release (currently version 3.1). I assume that's fine and to be expected, but wanted to clarify. Will we have any problems with version incompatibility? For example, METplus is running MET's develop branch which creates .stat file for MET version 10.0.0. But the latest stable version of METviewer (3.1) shouldn't know anything about MET version 10.0.0 since it pre-dates it.

Reply:
That is a good point to bring up. I noticed there is a develop image in the METviewer DockerHub. We should probably use this to ensure that we are using the latest of everything. This script also downloads the docker-compose.yml file from METviewer develop, so it would be good to be consistent.